packages/api/src/providers/tenovi.ts

```diff
@@ -91,11 +92,13 @@ export class Tenovi extends Provider {
       const url = `${Tenovi.URL}/${Tenovi.API_PATH}/hwi/unlink-gateway/${deviceId}/`;
 
       try {
-        await axios.get(url, {
-          headers: {
-            Authorization: `Api-Key ${Tenovi.apiKey}`,
-          },
-        });
+        if (deviceId !== TENOVI_TEST_DEVICE_ID) {
```

Comment:
NABD, but it would look cleaner to have the conditional right at the start of the function, so we don't execute anything if its a test device.

Reply:
The logic here is to not execute the axios request to de-activate the test device on Tenovi servers. But we still want to remove the test device from the user's providerMap. Although some refactoring is possible to not generate the URL.

Reply:
yeah, 💯 

I wrongly assumed all we did there was the call to the external API, good call